stdlib/3.4/asyncio/streams.pyi

```diff
@@ -12,9 +12,12 @@ _ClientConnectedCallback = Callable[[StreamReader, StreamWriter], Optional[Await
 __all__: List[str]
 
 class IncompleteReadError(EOFError):
-    def __init__(self, partial: str, expected: int) -> None: ...
+    expected = ...  # type: Optional[int]
+    partial = ...  # type: bytes
+    def __init__(self, partial: bytes, expected: int) -> None: ...
```

Comment:
Looks like the `expected` argument should also be `Optional[int]`.

Reply:
Good catch. I've pushed an update.